cpp/util/fake_etcd.cc

```diff
@@ -112,6 +112,9 @@ void FakeEtcdClient::Get(const Request& req, GetResponse* resp, Task* task) {
   CHECK(!req.recursive) << "not implemented";
   CHECK_LE(req.wait_index, 0) << "not implemented";
 
+  task->CleanupWhenDone(
```

Comment:
This was correct, but I figured this was a safer pattern, considering other changes coming here...